proto/autokitteh/projects/v1/svc.proto

```diff
@@ -104,6 +104,15 @@ message DownloadResourcesResponse {
   map<string, bytes> resources = 2;
 }
 
+message ExportRequest {
+  string project_id = 1 [(buf.validate.field).string.min_len = 1];
+}
+
+message ExportResponse {
+  // Zip of the mainfest + files
```

Comment:
i would just return the files unziped in a map with the manifest, but i that's fine as well.